actions/views/login.js

```diff
@@ -1,26 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import * as UserActions from 'mattermost-redux/actions/users';
-
-export function login(loginId, password, mfaToken) {
-    return (dispatch) => {
-        return ignoreMfaRequiredError(dispatch(UserActions.login(loginId, password, mfaToken)));
-    };
-}
-
-export function loginById(userId, password, mfaToken) {
-    return (dispatch) => {
-        return ignoreMfaRequiredError(dispatch(UserActions.loginById(userId, password, mfaToken)));
-    };
-}
-
-async function ignoreMfaRequiredError(promise) {
```

Comment:
enforced mfa error is never returned by login api, so this error doesnt need to be handled by this anyways